_docs/individual-payments.md

```diff
@@ -40,19 +40,19 @@ The valid `bank_code` values are:
 {% capture data-raw %}
 ```
 Access Bank: 044
+Citi Bank: 023
 EcoBank: 050
-FCMB Bank: 214
+FCMB : 214
 Fidelity Bank: 070
 First Bank of Nigeria: 011
 Guaranty Trust Bank : 058
 Heritage Bank: 030
 Jaiz Bank: 301
 Keystone: 082
 Polaris Bank: 076
-Stanbic IBTC Bank: 039
+Stanbic IBTC Bank: 221
```

Comment:
These don't just look documentation changes but might require development work if we need to support this within our providers. I'm also unsure about the code change for Stanbic. Why is it needed? Note that the codes we send to our customers don't need to correspond to codes that our payment partners use